packages/lesswrong/server/reviewVoteUpdate.ts

```diff
@@ -268,12 +252,12 @@ export async function createVotingPostHtml () {
     </tr>`
   }).join("")
 
-  return `<div class="votingResultsPost">
+  return `<div><div class="votingResultsPost">
     ${style}
     <table>
       ${postsHtml}
     </table>
-  </div>`
+  </div></div>`
```

Comment:
why extra div?

Reply:
If you don't have the extra div the html doesn't render in the post. I don't know why.